Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSource.java

```diff
@@ -83,14 +83,15 @@ private CardImageUrls innerGenerateURL(CardDownloadData card, boolean isToken) {
                 if (ScryfallImageSupportCards.isApiLink(link)) {
                     // api link - must prepare direct link
                     baseUrl = link + localizedCode + "?format=image";
-                    alternativeUrl = link + defaultCode + "?format=image";
-                    // workaround to use cards without english images (some promos or special cards)
-                    if (Objects.equals(baseUrl, alternativeUrl) && baseUrl.endsWith("/en?format=image")) {
-                        alternativeUrl = alternativeUrl.replace("/en?format=image", "?format=image");
-                    }
+                    // fallback no loc code
```

Comment:
Must keep original comments about real reason (why you must use links without loc code)

Reply:
Returned original workaround comment.